docs/tests/getting-started/github-actions.yml

```diff
@@ -23,9 +23,20 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
+      - name: Project init
+        uses: dagger/dagger-for-github@v2
+        # See all options at https://github.com/dagger/dagger-for-github
+        with:
```

Comment:
We should use the `version` property so that we know exactly which version of dagger our CI is going to run. By the way, I **think** that we support pinning to minors, so you can also use `0.2`.